src/main/PlagueMain.java

```diff
@@ -1529,7 +1530,7 @@ void loadMap(mindustry.maps.Map map) {
         });
         isSerpulo = PlagueData.serpuloCores.contains(Team.malis.cores().get(0).block);
         world.tiles.forEach(t -> {
-            if (t.build != null && t.build.block.equals(Blocks.powerSource))
+            if (t.build != null && t.build.block.equals(Blocks.powerSource) && t.team() == Team.malis)
```

Comment:
a Tile does not have a team() method, use `t.build.team()`

Reply:
it does but i changed it anyway